### PR TITLE
marqeta-client-business-requests: updated business requests for chang…

### PR DESCRIPTION
**Pull request by jbearden**
…e to fetch

This pull request is to adjust the Business Module requests.  Requests are now sanitized and snakecased in the index at `fetch()`.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -212,7 +212,13 @@ export function removeEmpty(obj: any): any {
  * must be removed before any update requests are sent.
  */
 export function cleanMetaData(obj: any): any {
-  const { createdTime, lastModifiedTime, ...ret } = obj // eslint-disable-line
+  /* eslint-disable */
+  const {
+    createdTime,
+    lastModifiedTime,
+    password,
+    ...ret } = obj // eslint-disable-line
```
Comment (drbobbeaty) [inline src/index.ts:220]: This ESLint disable for the line is unnecessary - it's bracketed by the others.

File: tests/business.test.ts [hunk 2]
```diff
@@ -39,7 +39,8 @@ import { Marqeta } from '../src';
   console.log('getting list of Businesses...')
   const list = await client.business.list()
 
-  if (list?.body?.isMore) {
+  if (list.body?.isMore) {
+    console.log(`[IS MORE]: ${JSON.stringify(list.body.isMore)}`)
```
Comment (drbobbeaty) [inline tests/business.test.ts:43]: This could look better... but it's up to you...

**Review by drbobbeaty — CHANGES_REQUESTED** [review]
a few really minor things...

File: tests/business.test.ts (hunk 2 above)
Comment (jbearden) [reply tests/business.test.ts:43]: nah, that was debug statement.  Removed.

**Review by drbobbeaty — APPROVED** [review]
looks good

**Pull request merged**